### ISSUE TITLE
"Invalid character detected." crash during parsing SVG

### ISSUE THREAD

**Issue by AndreyAfanasyev** [AngleSharp/AngleSharp#795]
### Steps to Reproduce
I have created a [MWE]:
```
[Test]
public void AngleSharpInvalidCharacterDetectedCrashTest()
{
	new HtmlParser().ParseDocument(
@"<html>
<body>
<svg>
	<g>
		<image xlink:href$="""" width=""274"" height=""186""></image>
	</g>
</svg>
</body>
</html>");
}
```

**Expected behavior:** Successful parsing of the html from AngleSharpInvalidCharacterDetectedCrashTest().

**Actual behavior:** AngleSharpInvalidCharacterDetectedCrashTest() crashes with "Invalid character detected." DomException.

**Environment details:** [Windows Server 2016, .Net 4.7.2]

**Comment by FlorianRappl** [comment]
Thanks for the report. Will be fixed in the upcoming 0.13 release!

**Comment by FlorianRappl** [comment]
Landed in `devel`!
